extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/KafkaProcessor.java

```diff
@@ -68,6 +76,9 @@
             ByteBufferDeserializer.class,
             StringDeserializer.class,
             FloatDeserializer.class,
+
+            //login modules
+            LoginModule.class
```

Comment:
This list was about serde, LoginModule might go in another list (it's actually not build-in as it comes from `javax.security.auth.spi.LoginModule`)

Reply:
Would a separate array/list suffice?